### ISSUE TITLE
Changing write speed to normal or high

### ISSUE THREAD

**Issue by laagabi** [NetAppDocs/occm#887]
Page: [Modifying Cloud Volumes ONTAP systems](https://docs.netapp.com/us-en/occm/task_modifying_ontap_cloud.html)

Looks like the option to change write speed was moved somewhere else. Could you please check it with engineering and update the documentation on the same?

https://i.imgur.com/0JRIuwG.png
![0JRIuwG](https://user-images.githubusercontent.com/80273701/159305255-e76b00b3-38ce-4d13-be1f-f0686b1caab7.png)

**Comment by netapp-bcammett** [comment]
Hey there,

Are you using a supported configuration? I still see the "Writing Speed" option:

![image](https://user-images.githubusercontent.com/18287750/159311430-94ba268a-9ef3-4f41-b7a1-a4587df1f795.png)

**Comment by laagabi** [comment]
Yep, I was running 9.8 with m5.2xlarge instance type (based on the doc<https://docs.netapp.com/us-en/occm/concept_write_speed.html#high-write-speed-with-an-ha-pair-in-aws> this should support HWS). Customer has a 9.10.1 cvo ha running in Azure and they can`t see this either.


***@***.***

Is your config recent or an older setup?

Gabor.
